MobileCenterCrashes/MobileCenterCrashes/Internals/Model/MSWrapperException.h

```diff
@@ -0,0 +1,26 @@
+#import <Foundation/Foundation.h>
+
+@class MSException;
+
+/**
+ * This class represents a wrapper exception that augments the data
+ * recorded when the application crashes.
+ */
+@interface MSWrapperException : NSObject 
+
+/**
+ * The model exception for the corresponding crash.
+ */
+@property(nonatomic, strong) MSException* modelException;
```

Comment:
strong is a default attribute. We decided not to specify default attributes for properties.